deluge/commonoptions.py

```diff
@@ -104,8 +105,8 @@ def __init__(self, *args, **kwargs):
                                 help="Set the log level: %s" % ", ".join(deluge.log.levels))
         self.group.add_argument("-q", "--quiet", action="store_true", default=False,
                                 help="Sets the log level to 'none', this is the same as `-L none`")
-        self.group.add_argument("-r", "--rotate-logs", action="store_true", default=False,
-                                help="Rotate logfiles.")
+        self.group.add_argument("-r", "--rotate-logs", action="store", nargs="?", const="50M",
+                                help="Rotate logfiles with optional max file size..")
```

Comment:
I think reword this "Enable logfile rotation (optional max file size, default 50MB)"

What do you think about renaming the command to `--logrotate`?